backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -116,13 +136,13 @@ public void deleteUser(UUID uuid) {
   }
 
   @Override
-  public UserDto getUser(UUID uuid) {
+  public UserResponse getUser(UUID uuid) {
```

Comment:
Wasn't this already refactored?

Reply:
It was, rebased by accident so it's showing in this PR too.